packages/path_provider/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.5.2
+
+* macOS support now allows for retrieving the user's downloads directory.
```

Comment:
nit: I woudn't mention macOS here, from this package's perspective a `getDownloadsDirectory` was added (which is not supported on Android and iOS)